src/coreclr/jit/emitfmtsloongarch64.h

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
If I understand correctly, the definitions in this file are not used anywhere except `emitInsSanityCheck` which is not called from your code. Remove this?

Reply:
> If I understand correctly, the definitions in this file are not used anywhere except `emitInsSanityCheck` which is not called from your code. Remove this?

Do you mean the `id->idInsFmt()` ?
It is just for compiling passed for some code within the DEBUG macro.